tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/SDKBuildPropertiesResourceListener.java

```diff
@@ -119,34 +156,25 @@ public void resourceChanged( final IResourceChangeEvent event )
         try
         {
             final IResourceDelta delta = event.getDelta();
-            final SDK sdk = SDKUtil.getWorkspaceSDK();
 
-            if( delta != null && sdk != null )
+            if( delta != null )
             {
                 for( IResourceDelta child : delta.getAffectedChildren() )
                 {
-                    final IPath deltaLocation = child.getResource().getLocation();
+                    final IResourceDelta[] sdkChangedFiles =
```

Comment:
because we merged the hasMoreThanOneSdkListener into sdkBuildProertiesListener, we need to handle the event that sdk project can be deleted. While this happened, the child.getResource will be return null and it will throw null exception, so I removed the sdk null checking. I want to make the resourceChange method can be mapped to two different listener check.